care/facility/tasks/soft_delete_assets.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+from celery import Celery
+
+from care.facility.models.asset import Asset
+from care.facility.models.facility import Facility
+
+broker_url = os.environ.get("CELERY_BROKER_URL")
+app = Celery("soft_delete_assets", broker=broker_url)
+
+
+@app.task
+def soft_delete_assets_schedule():
+    facilities = Facility.objects.all().values_list("external_id", flat=True)
+    Asset.objects.exclude(
+        current_location__facility__external_id__in=facilities
+    ).update(deleted=True)
```

Comment:
We could simply rely on the PK instead of the UUID right? @sainak 

```suggestion
def soft_delete_assets_schedule():
    facilities = Facility.objects.all().values_list("id", flat=True)
    Asset.objects.exclude(
        current_location__facility__id__in=facilities
    ).update(deleted=True)
```

Reply:
@rithviknishad we can do that also but as the filterset class also takes the external_id I considered external id just to avoid any existing wrong entry in the database to spoil the functionality otherwise we can do that

Reply:
use id as we try to stick with id for internal queries